Update interfaces/geolocation-API.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/d6ebeec/whatwg/idl/geolocation-API.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/583016313

foolip commented:
I think it's best to wait for https://github.com/w3c/geolocation-api/pulls to land before merging this, to not have tests requiring `GeolocationPosition` and `GeolocationCoordinates` to be exposed, when that'll soon change again.

autofoolip committed:
Update interfaces/geolocation-API.idl

Source: https://github.com/tidoust/reffy-reports/blob/d6ebeec/whatwg/idl/geolocation-API.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/583016313

foolip commented:
https://github.com/w3c/geolocation-api/pull/34 is reflected here now.

foolip commented:
Right after this I was going to send a PR to rename the test, but when backing up a step to fix the commit message of https://github.com/web-platform-tests/wpt/commit/bd4803e17f827b09d341457ca7a83b578d79f590 I accidentally committed directly to master in https://github.com/web-platform-tests/wpt/commit/c717f05c53ebd6c08d0800833674e36933d323e5.

It's not a very review-worthy change, but I'll check for regressions on master.
